code/graphics/opengl/gropenglstate.cpp

```diff
@@ -334,7 +332,7 @@ GLboolean opengl_state::ClipPlane(GLint num, GLint state)
 
 GLboolean opengl_state::ClipDistance(GLint num, GLint state)
 {
-	Assert( (num >= 0) || (num < (int)(sizeof(clipdistance_Status) / sizeof(GLboolean))) || GL_version >= 30 );
+	Assert( (num >= 0) && (num < (int)(sizeof(clipdistance_Status) / sizeof(GLboolean))) );
```

Comment:
I am pretty sure that the logical ORs were wrong here since this condition always evaluated to true (since `GL_version >= 30` was always true). Since this looks like a range check the logical ANDs seem to make a lot more sense.